raiden/transfer/node.py

```diff
@@ -0,0 +1,664 @@
+# -*- coding: utf-8 -*-
+from raiden.transfer import (
+    channel,
+    token_network,
+    views,
+)
+from raiden.transfer.mediated_transfer import (
+    initiator_manager,
+    mediator,
+    target,
+)
+from raiden.transfer.architecture import TransitionResult
+from raiden.transfer.state import (
+    NodeState,
+    PaymentMappingState,
+    PaymentNetworkState,
+)
+from raiden.transfer.state_change import (
+    ActionChangeNodeNetworkState,
+    ActionForTokenNetwork,
+    ActionInitNode,
+    ActionLeaveAllNetworks,
+    ActionNewTokenNetwork,
+    Block,
+    ContractReceiveChannelWithdraw,
+    ContractReceiveNewPaymentNetwork,
+    ContractReceiveNewTokenNetwork,
+    ReceiveUnlock,
+)
+from raiden.transfer.mediated_transfer.state_change import (
+    ActionInitInitiator,
+    ActionInitMediator,
+    ActionInitTarget,
+    ReceiveSecretRequest,
+    ReceiveSecretReveal,
+    ReceiveTransferRefund,
+    ReceiveTransferRefundCancelRoute,
+)
+
+
+def get_networks(node_state, payment_network_identifier, token_network_identifier):
+    token_network_state = None
+    payment_network_state = node_state.identifiers_to_paymentnetworks.get(
+        payment_network_identifier
+    )
+
+    if payment_network_state:
+        token_network_state = payment_network_state.tokenaddresses_to_tokennetworks.get(
+            token_network_identifier
+        )
+
+    return payment_network_state, token_network_state
+
+
+def get_token_network(node_state, payment_network_identifier, token_network_identifier):
```

Comment:
How is this different to the `get_token_network` function in `transfer/views.py`?

Reply:
at the moment, not different at all

Reply:
Will that change in the future?

Reply:
views are a different thing: https://github.com/raiden-network/raiden/pull/1288#issuecomment-370521424

the data returned by the views must be treated as immutable.

Reply:
anything else?